packages/salesforcedx-vscode-lwc/src/testSupport/types/index.ts

```diff
@@ -42,6 +42,15 @@ export const enum TestInfoKind {
   TEST_DIRECTORY = 'testDirectory'
 }
 
+/**
+ * Confirms if the TestExecutionInfo is specific to a testCase
+ */
+export function isTestCase(
```

Comment:
nit: being that you're type checking the `TestCaseInfo` type `isTestCaseInfo` seems like a better name.